setupTests.js

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { configure } from 'enzyme';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
Not a huge deal for the pr, but something to keep in mind is the longevity of enzyme. E.g., React 17 support has not landed yet: https://github.com/enzymejs/enzyme/issues/2429